dist/s_all

```diff
@@ -81,6 +81,9 @@ run "./s_clang-format"
 run "python prototypes.py"
 run "sh ./s_typedef -b"
 run "python test_tag.py"
+if command -v evergreen > /dev/null; then
+	run "evergreen --level notice validate ../test/evergreen.yml"
```

Comment:
What are the different levels ?

Reply:
As discussed:
```
--level value 
Specify lowest visible log level as string: 'emergency|alert|critical|error|warning|notice|info|debug|trace' (default: "info")
```

Reply:
With the `notice` level used, it does not print the current warning. We should leave it to the default value which works.